llvm/lib/Support/VirtualOutputBackends.cpp

```diff
@@ -356,7 +356,24 @@ Error OnDiskOutputFile::keep() {
   if (!RenameEC)
     return Error::success();
 
+  // Rename failed. Print some information for diagnosis.
```

Comment:
This deserves its own FIXME to remove direct printing to errs once we figure it out.

Should also print `RenameEC.message()` since it will be clobbered by the copy_file below and might not have the same error by the time we get a diagnostic, if any.